lib/version.sh

```diff
@@ -32,6 +33,21 @@ function find_latest_server_version() {
     echo "${server_version}"
 }
 
+function find_latest_ga_server_version() {
+    local server_version
+
+    # Note - we assume every release will have a enterprise component.
+    server_version="$(
+        fetch "${FUNCNAME[0]}" "${ARTIFACTS_DOMAIN}/aerospike-server-enterprise/" |
```

Comment:
Couldn't we extract the version from https://artifacts.aerospike.com/aerospike-server-enterprise/latest?

Seems better than relying on the ordering.